lib/PostBuildAction.js

```diff
@@ -1,144 +1,114 @@
- *
- * Permission is hereby granted, free of charge, to any person
- * obtaining a copy of this software and associated documentation
- * files (the "Software"), to deal in the Software without restriction,
- * including without limitation the rights to use, copy, modify, merge,
- * publish, distribute, sublicense, and/or sell copies of the Software,
- * and to permit persons to whom the Software is furnished to do so,
- * subject to the following conditions:
- *
- * The above copyright notice and this permission notice shall be included
- * in all copies or substantial portions of the Software.
- *
- * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS
- * OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
- * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
- * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
- * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
- * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
- * THE SOFTWARE.
- *
- * This license applies to all parts of catberry-assets that are not externally
- * maintained libraries.
- */
-
 'use strict';
 
-module.exports = PostBuildAction;
+const csstime = require('csstime-gulp-tasks');
+const prettyTime = require('pretty-hrtime');
+const util = require('util');
+const path = require('path');
+const gulp = require('gulp');
 
-var csstime = require('csstime-gulp-tasks'),
-	prettyTime = require('pretty-hrtime'),
-	util = require('util'),
-	path = require('path'),
-	gulp = require('gulp');
+const INFO_START = 'Starting assets building...';
+const TRACE_TASK_START = 'Starting task %s...';
+const TRACE_TASK_END = 'Task %s finished after %s';
+const ERROR_TASK = 'Task %s finished with error after %s: %s';
+const ERROR_TASK_NOT_FOUND = 'Task %s not found';
 
-var INFO_START = 'Starting assets building...',
-	TRACE_TASK_START = 'Starting task %s...',
-	TRACE_TASK_END = 'Task %s finished after %s',
-	ERROR_TASK = 'Task %s finished with error after %s: %s',
-	ERROR_TASK_NOT_FOUND = 'Task %s not found';
+class PostBuildAction {
 
-/**
- * Creates new instance of the assets post build action.
- * @param {Object} $config Catberry application config.
- * @param {Logger} $logger Logger.
- * @constructor
- */
-function PostBuildAction($config, $logger) {
-	this._isRelease = Boolean($config.isRelease);
-	this._config = $config.assets || {};
-	this._config.componentJSON = this._config.componentJSON ||
-		'cat-component.json';
-	this._config.destinationDir = this._config.destinationDir ||
-		$config.publicDirectoryPath || 'public';
-	this._config.destinationComponentsDir =
-		this._config.destinationComponentsDir || 'assets';
-	this._config.cdnPath = this._config.cdnPath || '/assets/';
-	this._logger = $logger;
-	this._wrapWithLogger(gulp);
-}
+	/**
+	 * Creates new instance of the assets post build action.
+	 * @param {Locator} locator
+	 * @constructor
```

Comment:
This jsDoc annotation is not necessary to use anymore. It's obvious that it's a constructor.